docs/build.gradle

```diff
@@ -35,15 +35,15 @@ buildRestTests.expectedUnconvertedCandidates = [
         'reference/ml/apis/update-snapshot.asciidoc',
 ]
 
-integTestCluster {
-  if ("default".equals(integTestCluster.distribution)) {
+testClusters.integTest {
+  if ("DEFAULT".equals(singleNode().distribution.name())) {
```

Comment:
We should probably just compare the enum values themselves rather than their string equivalents.